includes/class-wc-payments-account.php

```diff
@@ -981,6 +989,7 @@ function () {
 	 * @return array|bool|string Either the new account data or false if unavailable.
 	 */
 	public function refresh_account_data() {
+		$this->maybe_clear_payment_methods_cache();
```

Comment:
It makes sense to put it inside `WC_Payments_Customer_Service `. I will do that.

The reason why I put it here is that I wanted to avoid injecting `WC_Payments_Action_Scheduler_Service` inside `WC_Payments_Customer_Service ` and then inject that service inside `WC_Payments_Accounts` which will results in broken tests that needs to be fixed because constructor needs to be updated.